server/modelpath.go

```diff
@@ -106,6 +107,10 @@ func GetBlobsPath(digest string) (string, error) {
 		return "", err
 	}
 
+	if runtime.GOOS == "windows" {
+		digest = strings.ReplaceAll(digest, ":", "-")
```

Comment:
open to suggestions on how to name this in windows or in general

Reply:
I think this makes sense, I can't think of a better alternative